servicecontrol/db-compaction.md

```diff
@@ -7,80 +7,39 @@ tags:
 ---
 
 ## Overview
```

Comment:
@johnsimons @gbiellem - instead of removing the manual steps, do we keep them as a secondary option?
Also, do we need to mention the cases that @boblangley mentioned, i.e. for the esent utl to work SC service needs to be shutdown properly?